diff --git a/pyproject.toml b/pyproject.toml
index d5c44b9..17440ef 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -21,8 +21,8 @@ dependencies = [
     # Packages under the PyGfx umbrella.
     # For some of these we set an upper bound because (for now) they regularly break their API.
     # Packages that depend on PyGfx should not set an upper bound on e.g. wgpu, unless really needed!
-    "rendercanvas >= 2.2, !=2.3.0, !=2.4.0",
-    "wgpu >=0.24.0, <0.30",
+    "rendercanvas >= 2.6.1",
+    "wgpu >= 0.30.1",
     "pylinalg >=0.6.8,<0.7.0",
     # External dependencies
     "numpy",
